api/api_endpoints.py

```diff
@@ -0,0 +1,72 @@
+from flask import Blueprint, jsonify, request
+from app import db
+from app.base.models import User
+from api.schema import user_schema, users_schema
+
+api_blueprint = Blueprint("api_blueprint", __name__)
+
+
+@api_blueprint.route('/api/user/all')
+def get_users():
+    """ returns a json object of all the users in the database"""
+    users = User.query.all()
+    return users_schema.jsonify(users)
+
+
+@api_blueprint.route('/api/user/<int:user_id>', methods=['GET'])
+def get_one_user(user_id):
+    """ returns a json object of one user matching the id """
+    user = User.query.get(user_id)
+    if not user:
+        return jsonify({"message": "user not found"})
+    return user_schema.jsonify(user)
+
+
+@api_blueprint.route('/api/user', methods=['POST'])
+def add_user():
+    """ creates new user in the database with data from request body"""
```

Comment:
Start every docstring with an uppercase Letter and on the margin where the tripple quotes start from
and end the docstring with a full stop (dot)